premarum-backend/PreEnrollmentMgmt.WebApi/Controllers/PreEnrollmentController.cs

```diff
@@ -112,4 +113,12 @@ public async Task<IEnumerable<PreEnrollmentSemesterOfferDTO>> GetRecommendations
             preEnrollmentId, User.Identity?.Name!);
         return _mapper.Map<IEnumerable<PreEnrollmentSemesterOfferDTO>>(result);
     }
+
+    [Authorize]
+    [HttpGet("{preEnrollmentId}/Requisites")]

```

Comment:
Maybe include prerequisite in the route name to differentiate between the other route maybe implemented in the future for coreq.